jkube-kit/generator/api/src/test/java/org/eclipse/jkube/generator/api/support/BaseGeneratorTest.java

```diff
@@ -235,7 +229,7 @@ public void addFromInIsTagModeWithConfiguredImageAndSelectorShouldReturnConfigur
     // When
     new TestBaseGenerator(ctx, "test-generator", new TestFromSelector(ctx)).addFrom(builder);
     // Then
-    assertEquals("image:latest", builder.build().getFrom());
+    assertThat("image:latest").isEqualTo(builder.build().getFrom());
```

Comment:
The assertion should be inverted.
```java
assertThat(builder.build().getFrom()).isEqualTo("image:latest")
```
Even more, all of the assertions here could be combined into a single one.